resources/guest_configs/microvm-kernel-ci-x86_64-5.10-no-acpi.config

```diff
@@ -304,7 +313,6 @@ CONFIG_X86_MPPARSE=y
 # CONFIG_GOLDFISH is not set
 # CONFIG_X86_CPU_RESCTRL is not set
 # CONFIG_X86_EXTENDED_PLATFORM is not set
-# CONFIG_X86_AMD_PLATFORM_DEVICE is not set
```

Comment:
How come this goes away? Actually there are some lines like this. Is it because you built the kernel and took the .config?

Reply:
For this specific config I used  `make menuconfig` to disable ACPI, which then also removed a bunch of things depending on CONFIG_ACPI.